doc/index.rst

```diff
@@ -41,8 +41,6 @@ ODE-toolbox is written in Python and leverages SymPy for the symbolic manipulati
 Installation
 ------------
 
-.. Attention:: SymPy releases after 1.4 introduce a regression in runtime performance when computing matrix exponentials. Unless this conflicts with other version requirements, we recommend to use SymPy 1.4 for now (for example, by editing ``requirements.txt`` to read ``sympy==1.4``). This issue is being tracked at https://github.com/sympy/sympy/issues/23417.
```

Comment:
Technically, yes, but with this PR, the problem is avoided for most typical use cases (including all of those in the ODE-toolbox tests as well as all NESTML models). We could of course still leave a message in the documentation. Do you think it makes sense or would it just get lost in the noise?

Reply:
I think it is better to leave it there as the issue is still unresolved from the `sympy` end and also in case someone faces this regression in the future for some model.

Reply:
Could we raise this as an issue on the ODE-toolbox Github respository instead?

Reply:
Fine with me.

Reply:
Done! See #80.